_includes/pbc/all/install-features/202304.0/install-the-packaging-units-feature.md

```diff
@@ -0,0 +1,1098 @@
+
+
+**src/Pyz/Zed/ProductPackagingUnit/Persistence/Propel/Schema/spy_product_packaging_unit.schema.xml**
+
+```xml
+<?xml version="1.0"?>
+<database xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" name="zed" xsi:noNamespaceSchemaLocation="http://static.spryker.com/schema-01.xsd" namespace="Orm\Zed\ProductPackagingUnit\Persistence" package="src.Orm.Zed.ProductPackagingUnit.Persistence">
+
+    <table name="spy_product_packaging_unit" phpName="SpyProductPackagingUnit">
+        <behavior name="event">
+            <parameter name="spy_product_packaging_unit_all" column="*"/>
+        </behavior>
+    </table>
+
+    <table name="spy_product_packaging_unit_type" phpName="SpyProductPackagingUnitType">
+        <behavior name="event">
+            <parameter name="spy_product_packaging_unit_type_all" column="*"/>
+        </behavior>
+    </table>
+</database>
+```
+
+Set up synchronization queue pools, so non-multistore entities (not store-specific entities) can be synchronized among stores:
+
+**src/Pyz/Zed/ProductPackagingUnitStorage/Persistence/Propel/Schema/spy_product_packaging_unit_storage.schema.xml**
+
+```xml
+<?xml version="1.0"?>
+<database xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          name="zed"
+          xsi:noNamespaceSchemaLocation="http://static.spryker.com/schema-01.xsd"
+          namespace="Orm\Zed\ProductPackagingUnitStorage\Persistence"
+          package="src.Orm.Zed.ProductPackagingUnitStorage.Persistence">
+
+    <table name="spy_product_packaging_unit_storage">
+        <behavior name="synchronization">
+            <parameter name="queue_pool" value="synchronizationPool" />
+        </behavior>
+    </table>
+
+</database>
+```
+
+Run the following commands to apply the database changes and generate entity and transfer changes:
+
+```bash
+console propel:install
+console transfer:generate
+```
+
+{% info_block warningBox "Verification" %}
+
+Make sure that the following changes have been applied by checking your database:
+
+| DATABASE ENTITY                        | TYPE   | EVENT   |
+|----------------------------------------|--------|---------|
+| spy_product_packaging_unit             | table  | created |
+| spy_product_packaging_unit_type        | table  | created |
+| spy_product_abstract_packaging_storage | table  | created |
+| spy_sales_order_item.amount            | column | created |
+| spy_sales_order_item.amount_sku        | column | created |
+
+{% endinfo_block %}
+
+{% info_block warningBox "Verification" %}
+
+Make sure that the following changes in transfer objects have been applied:
+
+| TRANSFER                                     | TYPE     | EVENT   | PATH                                                                       |
+|----------------------------------------------|----------|---------|----------------------------------------------------------------------------|
```

Comment:
We have to extend the list based on the current state of the modules:
* ProductPackagingUnit
* ProductPackagingUnitsBackendApi